src/app/checkout/accordion.controller.js

```diff
@@ -102,10 +104,18 @@ angular.module('checkoutModule')
                 getAddresses();
             });
 
+            checkoutService.loadSavedCC().then(function( res ){
+                if (res !== null){
+                    savedCC = res;
+                }
+            });
 
             // Load payment methods
             checkoutService.loadPaymentMethods().then(function(methods){
                 $scope.paymentMethods = methods;
+
+                //merge payment methods and saved Credit Cards;
+                $scope.paymentMethods = $scope.paymentMethods.concat(savedCC);
```

Comment:
I think we can't be sure that payments will be loaded after saved cc tokens. So what if payments are loaded and tokens are not yet? My suggestion is to concatenate `methods` and `savedCC` with `$scope.paymentMethods`